packages/core/src/test/shared/wizards/prompterTester.ts

```diff
@@ -24,38 +24,87 @@ export class PrompterTester {
 
     handleQuickPick(titlePattern: string, handler: (input: TestQuickPick) => void): PrompterTester {
         this.quickPickHandlers.set(titlePattern, handler)
+        this.handlerCallHistory.set(titlePattern, 0)
         return this
     }
 
     handleInputBox(titlePattern: string, handler: (input: TestInputBox) => void): PrompterTester {
         this.inputBoxHanlder.set(titlePattern, handler)
+        this.handlerCallHistory.set(titlePattern, 0)
         return this
     }
 
-    build(): void {
+    build(): PrompterTester {
         this.testWindow.onDidShowQuickPick((input) => {
             return this.handle(input, this.quickPickHandlers)
         })
         this.testWindow.onDidShowInputBox((input) => {
             return this.handle(input, this.inputBoxHanlder)
         })
+        return this
     }
 
     private record(title: string): void {
-        this.calledOrder++
-        this.report.set(title, this.calledOrder)
+        this.callLog.push(title)
+        this.handlerCallHistory.set(title, (this.handlerCallHistory.get(title) ?? 0) + 1)
+    }
+
+    /**
+     * Asserts that a specific handler has been called the expected number of times.
+     *
+     * @param title - The title or identifier of the handler to check.
+     * @param expectedCall - The expected number of times the handler should have been called.
+     * @throws AssertionError if the actual number of calls doesn't match the expected number.
+     */
+    assertHandlerCall(title: string, expectedCall: number) {
```

Comment:
why is "handler" mentioned here but not in `assertCallOrder` ? 
```suggestion
    assertCall(title: string, expectedCall: number) {
```

Reply:
Acknowledge. I'll update this.